doc/architecture/context.md

```diff
@@ -12,7 +12,12 @@ Global value and local values in the Soil layer:
   - 4 heavy (sT, L, uL) (German: schwer)
   - 5 very heavy (T, IT) (German: sehr schwer)
 - pH base value for topsoil from 0-14
-  (German: Bodenreaktion, e.g. Kohl basisch, ph3 Heidelbeeren), dynamisch
+  (German: Bodenreaktion, e.g. Kohl basisch, ph3 Heidelbeeren), dynamisch:
+  - 1 very acid (pH 5.0 and below)
+  - 2 acid (pH 6.0 - 6.5)
+  - 3 neutral (pH 6.6 - 7.3)
+  - 4 alkaline (pH 7.4 - 7.8)
+  - 5 very alkaline (pH 7.9 and above)
 - pH base value for subsoil from 0-14
 - yield grade (German: Ages Gehaltsklasse (Nährstoffversorgung und Humusgehalt) ÖNORM L 1210, determined from NPK, eigentlich für Intensivkultur, Stark/Schwachzehrer)
   from:
```

Comment:
Please also update the other stuff in this file, including everything in #45. (If it is not MISSING anymore, remove the word MISSING).